database.go

```diff
@@ -140,6 +144,7 @@ func initDatabase() {
 	db, err = sql.Open("sqlite3", "./database.sqlite")
 	checkErr(err)
 	createDatabase()
-	addUser("John", "Smith", "jonh.smith@example.com", "jsmith", "123")
+	addUser("John", "Smith", "jonh.smith@example.com", "jsmith", "123", "normal")
+	addUser("Nicoliere", "Hacedor de Sombreros", "n.h@example.com", "hnicoliere", "123", "root")
```

Comment:
Please use made up names for now.
```suggestion
	addUser("Satoshi", "Nakamoto", "snakamoto@example.com", "root", "123", "root")
```

Reply:
Ok, the new user has been changed to a made up name